src/pykeen/training/lcwa.py

```diff
@@ -71,10 +71,7 @@ def _label_loss_helper(
                 num_classes=self.model.num_entities,
             )
 
-        return self.model._compute_loss(
-            tensor_1=predictions,
-            tensor_2=labels,
-        )
+        return self.model.compute_label_loss(predictions, labels)
```

Comment:
While the `compute_label_loss` is only a thin wrapper around `_compute_loss`, this needs to be checked for semantic correctness